src/thinkspace/client/thinkspace-builder-rat/addon/components/confirmation/details.coffee

```diff
@@ -0,0 +1,29 @@
+import ember from 'ember'
+import base  from 'thinkspace-base/components/base'
+
+###
+# # details.coffee
+- Type: **Component**
+- Package: **ethinkspace-builder-pe**
+###
+export default base.extend
+  builder: ember.inject.service()
+
+  model: ember.computed.reads 'builder.model'
+  step:  ember.computed.reads 'builder.step_details'
+
+  init_base: ->
+    @get('step').validate().then =>
+      @set('changeset', @get('step.changeset'))
+
+  has_title_error: ember.computed 'changeset', ->
```

Comment:
I might be off here, but I think you can just do `if changeset.errors.title` in the template and remove this computed property.  The logic at the end gets a bit mind bending.  This would be the same for `has_instruction_error`, etc, too.   

```
    if changeset.errors.instructions
      br
      component '__changeset/errors' changeset=step.changeset attribute='instructions' errors_visible=true
    else
      = model.instructions
```